### PR TITLE
Add support for Active Job and its assertions

### PR DESCRIPTION
**Pull request by liseki**
Catching up with Active Job support in Rails 4.2.

### CODE REVIEW

File: lib/minitest/rails/expectations.rb [hunk 1]
```diff
@@ -530,6 +532,182 @@ module Minitest::Rails::Expectations
...
+    #
+    # :method: must_perform_with
+    # :call-seq: must_perform_with(args)
+
+
+    # This exists as a module to allow easy mixing into classes
+    # other than ActiveJob::TestCase where you might want to do
+    # job testing e.g. in an Active Record model which triggers
+    # jobs in a callback.
+    module ActiveJob
```
Comment (blowmage) [inline lib/minitest/rails/expectations.rb]: How about aliasing these methods on `ActiveJob::TestHelper` instead of creating a new module?

Comment (liseki) [inline lib/minitest/rails/expectations.rb]: That would be much cleaner but unfortunately ```ActiveJob::TestHelper``` has all its methods wrapped within an ```ActiveSupport::Concern#included``` block which essentially hides them. Looking at the module only the ```delegate``` call required wrapping so I guess it is an instance of rails-ism when ruby-ism would have been perfect.

Comment (blowmage) [inline lib/minitest/rails/expectations.rb]: > I guess it is an instance of rails-ism when ruby-ism would have been perfect.

Well said.

**Comment by blowmage** [comment]
Thanks you very much for your contribution!

**Comment by liseki** [comment]
It's a pleasure. Thank you for the gem!

**Pull request merged**